### ISSUE TITLE
train

### ISSUE THREAD

**Issue by CD-1998** [ArdhenduBehera/cap#4]
The basic_schedule in main.py is not used, that is ok? and your proposal learningrate is 1e-6 is initial? I should give more train epochs ?

**Comment by ArdhenduBehera** [comment]
Yes, the code is not cleaned properly.
Try with default learning rate of 1e-4.

**Comment by CD-1998** [comment]
Thank your reply. Recently，I have another question: as your paper said，different dataset corresponds different backbone, and should different backbone correspond different preprocess_input in keras? In fact, the resnet50 backbone and its preprocess_input can't get 94.9% acc when I use airplane dataset, and the resnet50 is lower than xception. Should different should use different learning rate ?

**Comment by ArdhenduBehera** [comment]
Others have reproduced the results (see the thread "How can I reproduce the result?")

Merge train and validation dataset to train the model, and test on test images.
Set batch size 4
Start learning rate 1e-4 and multiply 0.1 at every 50 epochs. Train for 200 epochs.
With this setting, we got best accuracy around 190 epochs.

**Comment by CD-1998** [comment]
Thanks for your answer,I ignore batch_size and i will try again. Iearning-rate in all dataset is same. Can you share batch_size in others ?
